concourse/pipelines/create-deployer.yml

```diff
@@ -514,6 +519,29 @@ jobs:
         params:
           file: concourse-deploy/concourse-manifest-state.json
 
+    - task: add-env-specific-team
+      config:
+        image: docker:///governmentpaas/self-update-pipelines
+        inputs:
+          - name: paas-cf
+          - name: concourse-manifest
+        params:
+          SYSTEM_DNS_ZONE_NAME: {{system_dns_zone_name}}
+          CONCOURSE_ATC_PASSWORD: {{concourse_atc_password}}
+          CONCOURSE_ATC_USER: admin
+          FLY_TARGET: {{deploy_env}}
+          FLY_CMD: ./paas-cf/bin/fly
+        run:
+          path: sh
+          args:
+            - -e
+            - -c
+            - |
+              export CONCOURSE_URL="https://deployer.${SYSTEM_DNS_ZONE_NAME}"
+              ./paas-cf/concourse/scripts/fly_sync_and_login.sh
+              echo y | ./paas-cf/bin/fly -t {{deploy_env}} set-team -n {{aws_account}} \
```

Comment:
Again, I disagree. 

It is not so much code, it is only one character more. And I insist as commented in https://github.com/alphagov/paas-cf/pull/546/files#r83616802 it makes sense to keep consistency, this is inconsistent with the task itself and the rest of the pipeline.

Also, we try to use environment variables in these cases, as the code can be extracted to a external script or a external task definition without big changes.